### PR TITLE
Pset2

### PR DESCRIPTION
**Pull request by Tamila-Bodnar**
Добавила задания Pset0, Pset1, Pset2

### CODE REVIEW

File: pset_2/result.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (AlexOrd) [inline pset_2/result.txt]: empty line

Comment (Tamila-Bodnar) [reply pset_2/result.txt]: Maybe I wrong pushing the files, because in my PC all right (result.txt wasn't be empty)

**Comment by Tamila-Bodnar** [comment]
Done @AlexOrd

**Comment by Tamila-Bodnar** [comment]
https://github.com/Tamila-Bodnar/Javascript/blob/pset2/pset_2/result.txt

**Pull request merged**